setup.json

```
@@ -163,6 +163,7 @@
             "numeric = aiida.orm.nodes.data.numeric:NumericType",
             "orbital = aiida.orm.nodes.data.orbital:OrbitalData",
             "remote = aiida.orm.nodes.data.remote:RemoteData",
```

While field testing, I got this error:

```
aiida.common.exceptions.StoringNotAllowed: class `aiida.orm.nodes.data.remote.base:RemoteData` does not have registered entry point
```

I'm assuming since this needs needs to be changed to:

```suggestion
            "remote = aiida.orm.nodes.data.remote.base:RemoteData",
```

---

Hmm, I thought that would have worked since it is also importable from `aiida.orm.nodes.data.remote`. But apparently the entry point needs the direct path where it is defined. Will change it.